webpack/webpack.config.dev-client.js

```diff
@@ -84,11 +84,10 @@ module.exports = {
       publicPath: publicPath
     },
     module: {
-      loaders: commonLoaders.concat([
-        { test: /\.css$/,
-          loader: 'style!css?module&localIdentName=[name]__[local]___[hash:base64:5]!postcss-loader'
-        }
-      ])
+      loaders: commonLoaders.concat({
+        test: /\.css$/,
+        loader: 'style!css?module&localIdentName=[name]__[local]___[hash:base64:5]!postcss-loader'
```

Comment:
Also I found that official document says we cannot use **css?modules** with **postcss-loader**. To use both of them, we should change it into **css?modules&importLoaders=1**
[Link}(https://github.com/postcss/postcss-loader#css-modules)